oak_runtime/src/proxy.rs

```diff
@@ -313,6 +313,17 @@ impl RuntimeProxy {
         result
     }
 
+    /// Calls [`Runtime::handle_clone`] using the Node's privilege.
```

Comment:
nit: drop "using the Node's privilege" &ndash; I don't think any privilege is needed.